### PR TITLE
Add Validation and Parsing to Forms

### PR DESCRIPTION
**Pull request by qwwqwwq**
- Upgrades JQuery / Lodash libraries
- Adds moment js angular wrapper
- Adds bootstrap CSS

### CODE REVIEW

**Comment by qwwqwwq** [comment]
Adding bootstrap CSS kinda broke some of the other stylings, let me know if you think its fixable, if  its to hard we can remove and pursue something else. Bootstrap is really useful for form elements so I think it would be worth the effort if we can have it.

File: winterfell/webapp/src/js/app.js [hunk 1]
```diff
@@ -4,54 +4,57 @@
...
     })
 
     .when("/form/:formId", {
-        templateUrl: "templates/form.html"
+      templateUrl: "templates/form.html"
     })
 
     // Otherwise
     .otherwise({
-        redirectTo: '/'
```
Comment (ahsu1230) [inline winterfell/webapp/src/js/app.js]: woops thanks for all the indentation fixes

File: winterfell/webapp/src/js/formCtrl.js [hunk 2]
```diff
@@ -20,10 +24,18 @@ app.controller("formCtrl", ['$scope', 'formRenderingService', 'formTemplateServi
...
+        $scope.model = {};
         $scope.fields = formTemplateService[$routeParams.formId];
 
-        $scope.model = {};
+        for (var i=0; i<$scope.fields.length; i++) {
+            if ($scope.fields[i].key.indexOf('date_signed') !== -1) {
+                $scope.model[$scope.fields[i].key] = currentDate();
+            }
+        }
     }]);
```
Comment (ahsu1230) [inline winterfell/webapp/src/js/formCtrl.js]: a real small nit, but just fyi: let's try to end all files with a new line.
http://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

**Comment by ahsu1230** [comment]
lgtm

File: winterfell/webapp/src/js/formCtrl.js (hunk 2 above)
Comment (qwwqwwq) [inline winterfell/webapp/src/js/formCtrl.js]: agreed, will setup my ide to do this

**Pull request merged**